dotcom-rendering/src/components/ListenToArticle.importable.tsx

```diff
@@ -30,23 +33,45 @@ export const ListenToArticle = ({ articleId }: Props) => {
 		number | undefined
 	>(undefined);
 
-	const isBridgetCompatible = useIsBridgetCompatible('8.6.0');
+	const isBridgetCompatible = useIsBridgetCompatible('8.7.0');
 	useEffect(() => {
 		if (isBridgetCompatible) {
 			Promise.all([
+				// AB TESTING native
+				getNativeABTestingClient().getParticipations(),
 				getListenToArticleClient().isAvailable(articleId),
 				getListenToArticleClient().isPlaying(articleId),
 				getListenToArticleClient().getAudioDurationSeconds(articleId),
 			])
-				.then(([isAvailable, isPlaying, durationSeconds]) => {
-					setAudioDurationSeconds(
-						typeof durationSeconds === 'number' &&
-							durationSeconds > 0
-							? durationSeconds
-							: undefined,
-					);
-					setShowButton(isAvailable && !isPlaying);
-				})
+				.then(
+					([
+						AbParticipations,
+						isAvailable,
+						isPlaying,
+						durationSeconds,
+					]) => {
+						// AB TESTING native start
+						const variant = AbParticipations.get(
+							'l2a_article_button_test',
+						);
+						if (variant === 'no-button') {
+							setShowButton(false);
+						}
+						if (variant === 'with-duration') {
+							setAudioDurationSeconds(
+								typeof durationSeconds === 'number' &&
+									durationSeconds > 0
+									? durationSeconds
+									: undefined,
+							);
+							setShowButton(isAvailable && !isPlaying);
+						}
+						if (variant === 'without-duration') {
```

Comment:
Another small thing, but these could be written with `else if` clauses, just for clarity and to avoid evaluating the extra conditions unnecessarily:

```suggestion
						if (variant === 'no-button') {
							setShowButton(false);
						} else if (variant === 'with-duration') {
							setAudioDurationSeconds(
								typeof durationSeconds === 'number' &&
									durationSeconds > 0
									? durationSeconds
									: undefined,
							);
							setShowButton(isAvailable && !isPlaying);
						} else if (variant === 'without-duration') {
```